docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -93,9 +96,39 @@ To open and close signals, either:
 *Close/Open selected*.
 
 [float]
+[[signals-to-timelines]]
 === Send signals to the Timeline
 
-To investigate a signal in the Timeline, click the *View in timeline* icon.
+To investigate a signal in the Timeline, click the *Investigate in timeline*
+icon.
+
+If the rule that generated the signal uses a timeline template, when you send
+the signal to the Timeline, the following dropzone query values are replaced
+with their corresponding signal values:
+
+* `host.name`
+* `host.hostname`
+* `host.domain`
+* `host.id`
+* `host.ip`
+* `client.ip`
+* `destination.ip`
+* `server.ip`
+* `source.ip`
+* `network.community_id`
+* `user.name`
+* `process.name`
+
+*Example*
+
+The timeline template used in the rule has this query:
```

Comment:
replace "query" with "dropzone query".  (I know it seems overly detailed, but to avoid confusion with Saved Queries or KQL queries, I think we should repeat "dropzone" each time.)